notifications/README.md

```diff
@@ -0,0 +1,58 @@
+# Notifications
+
+## Notification Object
+
+```
+{
+  id: string,
```

Comment:
This is automatic by firestore, right?

Reply:
Yes, but to send that in the response object or not, that depends on us. Should we not include it in our response?

Reply:
Are there any benefits you can think of?

Reply:
Not really

Reply:
Think some more :)
Ask the others.

Reply:
Done.